low-level/files/opt/seagate/sspl/setup/resource_map/storage.py

```diff
@@ -158,6 +159,42 @@ def get_psu_info(self):
             data.append(psu_dict)
         return data
 
+    def get_platform_sensors(self):
+        sensor_list = ['temperature', 'current', 'voltage']
+        sensor_data = self.build_encl_platform_sensors_data(sensor_list)
+        return sensor_data
+
+    def build_encl_platform_sensors_data(self, platform_sensors):
+        sensors_data = {}
+        sensors_resp = self.get_realstor_encl_data('sensor-status')
+        if sensors_resp:
+            for platform_sensor in platform_sensors:
+                for sensor in sensors_resp['api-response']['sensors']:
+                    if sensor['sensor-type'].lower() == platform_sensor:
+                        single_sensor_data = {
+                          'uid': sensor.get('durable-id'),
+                          'fru': 'false',
+                          'last_updated':  int(time.time()),
+                          'health': {
+                            'status': sensor.get('status'),
+                            'description': '',
+                            'recommendation': '',
```

Comment:
Left these values as empty string? Can you use below?
```
'description': sensor.get("description", "NA"),
'recommendation': sensor.get("recommendation", "NA")  
```

Reply:
Done.